dev-docs/dev.md

```diff
@@ -550,132 +528,83 @@ See [Create a GCE Windows test VM](create-gce-windows-test-vm.md).
 
-    ```
+    ```powershell
     sc delete google-cloud-ops-agent-fluent-bit
     sc delete google-cloud-ops-agent-otel
     sc delete google-cloud-ops-agent
     rd -r "C:\ProgramData\Google\Cloud Operations\Ops Agent"
     rd -r "C:\Program Files\Google\Cloud Operations\Ops Agent"
     ```
+    </details>
 
-1.  Verify that the service is indeed running.
-
-    ```
-    Get-Service google-cloud-ops-agent*
-    ```
+3.  [Verify](#windows-verify-status) that the service is indeed running.
 
     <details>
-    <summary>Expected output:</summary>
+    <summary>Troubleshooting</summary>
 
-    ```
-    Status   Name                               DisplayName
-    ------   ----                               -----------
-    Running  google-cloud-ops-agent             Google Cloud Ops Agent
-    Running  google-cloud-ops-agent-fluent-bit  Google Cloud Ops Agent - Logging Agent
-    Running  google-cloud-ops-agent-otel        Google Cloud Ops Agent - Metrics Agent
-    ```
+    -   Check logs in `Event Viewer` app
+    -   Check service status and start / stop them in `Services` app
+    -   Inspect running processes in `Task Manager` app
+    -   [Run Fluent Bit individually](#run-fluent-bit-individually-on-windows
+    ) or [run Open Telemetry Metrics Agent individually](#run-open-telemetry-metrics-agent-individually-on-windows).
 
     </details>
 
-**Troubleshooting**
-
--   Check logs in `Event Viewer` app
--   Check service status and start / stop them in `Services` app
--   Inspect running processes in `Task Manager` app
--   Run Fluent Bit individually:
-
-    Temporary step: Manually create a `C:\ProgramData\Google\Cloud
-    Operations\Ops Agent\run\buffers` folder. `C:\ProgramData` is by default
-    hidden, you need to show it.
-
-    ![image](https://user-images.githubusercontent.com/5287526/133006710-2ab21ef6-22e7-43ff-826d-15e82a2d3e08.png)
-
-
-    ```
-    C:\Users\{{USERNAME}}\tmp\out\bin\fluent-bit.exe --config="C:\ProgramData\Google\Cloud Operations\Ops Agent\generated_configs\fluentbit\fluent_bit_main.conf" --parser="C:\ProgramData\Google\Cloud Operations\Ops Agent\generated_configs\fluentbit\fluent_bit_parser.conf" --storage_path="C:\ProgramData\Google\Cloud Operations\Ops Agent\run\buffers"
-    ```
-
--   Run Open Telemetry Metrics Agent individually:
-
-    ```
-    C:\Users\{{USERNAME}}\tmp\out\bin\google-cloud-metrics-agent_windows_amd64.exe "--config=C:\ProgramData\Google\Cloud Operations\Ops Agent\generated_configs\otel\otel.yaml"
-    ```
-
-#### Uninstall the agent
-
-##### Linux
-
-1.  Stop the agent:
-
-    ```shell
-    sudo service google-cloud-ops-agent stop
-    ```
-
-1.  Uninstall the package:
-
-    ```shell
-    sudo apt remove google-cloud-ops-agent
-    ```
-
-##### Windows
-
-1.  Run the following commands in Windows PowerShell (admin mode):
-
-    ```
-    googet -noconfirm remove google-cloud-ops-agent
-    ```
 
-#### [Optional] Edit config
+## Edit config and apply
```

Comment:
Agree - I think I need to point out that this step is optional. 

This section used to be a subsection of `build on GCE`. I thought I should separate this to its own section because editing config should be a common task, not just for building on GCE. 

Now I have added a note in this section to mention that editing config is optional.